modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc

```diff
@@ -6,27 +6,23 @@ See also the complementary step on the last line of this file.
 
 ifdef::context[:parent-context: {context}]
 
+
+////
+Metadata attribute that will help enable correct parsing and conversion to the appropriate DITA element.
+////
+:_mod-docs-content-type: ASSEMBLY
+
 ////
  Base the file name and the ID on the assembly title. For example:
 * file name: assembly-my-user-story.adoc
 * ID: [id="assembly-my-user-story_{context}"]
 * Title: = My user story
-////
-
-////
-Indicate the content type in one of the following
-ways:
-Add the prefix assembly- or assembly_ to the file name.
-Add the following attribute before the module ID:
-:_mod-docs-content-type: ASSEMBLY
-////
 
-////
 The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken. Include {context} in the ID so the assembly can be reused.
 ////
-
 [id="assembly-my-user-story_{context}"]
 = My user story
+
 ////
 Be sure to include a line break between the title and the :context: variable and the :context: variable and the assembly introduction.
 
```

Comment:
The following line is the first time the instructions refer to lines above them. I find this confusing, it would make more sense to me to move this information to the comment above the ID and section title.

Reply:
Similarly, the explanation of the `:context:` attribute would make more sense in this place, above the actual `:context:` attribute definition.

Reply:
And as GitHub won't let me comment on those lines, all instructions regarding include statements would make more sense above the include statements, including the one about the empty line in between them. It would allow you to demonstrate that principle clearly without the includes being consistently interrupted with additional information.

Reply:
Done.